### DISCUSSION TITLE
Supply should support uploading Native debugging symbols

### DISCUSSION THREAD

**Discussion by rampr** [fastlane/fastlane#17270] [Ideas] [answered]
Google Api allows uploading proguard and naive debugging symbols as mentioned in https://developers.google.com/android-publisher/api-ref/rest/v3/edits.deobfuscationfiles

Right now the implementation only supports uploading proguard mappings -> https://github.com/fastlane/fastlane/blob/master/supply/lib/supply/client.rb#L335

Might be a very easy change - need to create a mapping type and support both proguard and native debug symbols. Happy to send a PR if that makes sense.

Please share your thoughts. Thanks in advance.

**Comment by PaulWoitaschek** [comment]
> Please share your thoughts. Thanks in advance.

I don't think there is any argument against this.

**Comment by rogerluan** [comment, marked as answer]
This was just released here https://github.com/fastlane/fastlane/pull/18526 🎉
